src/common/camera_control.c

```diff
@@ -1013,13 +1013,20 @@ void *dt_update_cameras_thread(void *ptr)
     if(camctl)
     {
       if((camctl->import_ui == FALSE) && (cv && (cv->view(cv) == DT_VIEW_LIGHTTABLE)))
+//TODO:  && import module is expanded and visible
```

Comment:
Yes, probably a good idea indeed, so it should be something like:

`dtgtk_expander_get_expanded(<import_module>->expander);`

But right here it is far from trivial as we don't have the import lib module. One solution would be to scan all the libs from darktable.lib GList, looking for "import" and checking the expander. Not sure we want to do that again and again.

So maybe store in main darktable struct a boolean `import_expanded` which is true when import is expanded? Anyway, no clean solution...

Reply:
Yeah, I figured we'll probably need to set up a variable that is updated by the import module itself as it goes in and out of visibility.

Reply:
Do you want to do that as part of this PR or another one?

Reply:
Let's do that separately.